lib/anemone/extractors/dirbuster.rb

```diff
@@ -6,7 +6,7 @@ def run
     @@dirs ||= nil
 
     return @@dirs if @@dirs
-    @@dirs = IO.read( File.dirname( __FILE__ ) + '/dirbuster/directories' ).split( "\n" )
+    @@dirs = File.binread( File.dirname( __FILE__ ) + '/dirbuster/directories' ).lines
```

Comment:
I believe this is semantically different 👀

Reply:
This new approach will contain `\n` characters:

```
3.0.2 :002 > File.binread("./Gemfile.lock").lines
 => 
["PATH\n",
 "  remote: .\n",
```

I believe we want `lines(chomp: true)`:
```
3.0.2 :003 > File.binread("./Gemfile.lock").lines(chomp: true)
 => 
["PATH",
 "  remote: .",
 "  specs:",
 "    metasploit-framework (6.1.33)",
 "      actionpack (~> 6.0)",
```

Was it not possible to use IO.readlines in the end? 👀 

https://ruby-doc.org/core-2.5.0/IO.html#method-c-readlines

Reply:
Some of the existing libraries already make use of `IO.readlines`. I think that is ok - it's IO.read that is a problem - but I haven't tested.